pkg/controller/certificates/doc.go

```diff
@@ -1,19 +0,0 @@
-/*
-Copyright 2016 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-// Package certificates contains logic for watching and synchronizing
-// CertificateSigningRequests.
```

Comment:
With this file gone, and a lot of the code being moved into two new packages, they could each use their own package-level godoc (either inline in an existing file, or a `doc.go` file).